crates/sui-json-rpc-types/src/lib.rs

```diff
@@ -1870,6 +1870,10 @@ pub struct SuiTransactionEffects {
     // The status of the execution
     pub status: SuiExecutionStatus,
     pub gas_used: SuiGasCostSummary,
+    // The version that every modified (mutated or deleted) object had before it was modified by
+    // this transaction.
+    #[serde(default, skip_serializing_if = "Vec::is_empty")]
+    pub modified_at_versions: Vec<(ObjectID, SequenceNumber)>,
```

Comment:
This style, the pair `(ObjectID, SequenceNumber)`, does not seem in line with the rest of this struct, but I was unsure if I should make a type just for this case. 

Any advice @patrickkuo?

Reply:
We use custom type (e.g. `SuiObjectRef`) to give the values names before serialisation to make it more readable.

IIRC this is preferred in typescript? CC @666lcz for comments